### PR TITLE
Initial application structure

### PR DESCRIPTION
**Pull request by PhilDigitalJustice**

### CODE REVIEW

File: .github/workflows/on-tag.yml [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+          APP_VERSION: ${{ needs.build-test-publish.outputs.app_version }}
+        run: |
+          echo "Captured version: ${{ env.APP_VERSION }}"
+          docker build --build-arg app_version=${{ env.APP_VERSION }} -t $REGISTRY/$REPOSITORY:${{ env.REPO_NAME }}-${{ env.APP_VERSION }} .
+          docker push $REGISTRY/$REPOSITORY:${{ env.REPO_NAME }}-${{ env.APP_VERSION }}
+
+      - name: delete artifact
+        uses: geekyeggo/delete-artifact@v2
+        with:
+          name: caab-api-jar
```
Comment (porritta) [inline .github/workflows/on-tag.yml:119]: assessment-api-jar

File: .github/workflows/on-tag.yml [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+      id-token: write # for requesting jwt
+      contents: read  # for actions/checkout
+    steps:
+      # Checkout github repo
+      - uses: actions/checkout@v3
+      # Assume role in cloud platform
+      - name: download jar
+        uses: actions/download-artifact@v3
+        with:
+          name: caab-api-jar
```
Comment (porritta) [inline .github/workflows/on-tag.yml:92]: assessment-api-jar

File: .github/workflows/on-tag.yml (hunk 1 above)
Comment (PhilDigitalJustice) [reply .github/workflows/on-tag.yml]: amended

File: .github/workflows/on-tag.yml (hunk 2 above)
Comment (PhilDigitalJustice) [reply .github/workflows/on-tag.yml]: amended

**Review by porritta — APPROVED** [review]
LGTM

**Pull request merged**